Angular rc2 breaks dependencies

When trying to follow Angular 2.0 Meteor Tutorial I get an error when executing:
meteor npm install --save angular2-meteor @angular/platform-browser-dynamic

...
npm ERR! peerinvalid The package @angular/core@2.0.0-rc.2 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer @angular/common@2.0.0-rc.2 wants @angular/core@2.0.0-rc.2
npm ERR! peerinvalid Peer @angular/compiler@2.0.0-rc.2 wants @angular/core@2.0.0-rc.2
npm ERR! peerinvalid Peer @angular/platform-browser@2.0.0-rc.2 wants @angular/core@2.0.0-rc.2
npm ERR! peerinvalid Peer @angular/platform-browser-dynamic@2.0.0-rc.2 wants @angular/core@2.0.0-rc.2
npm ERR! peerinvalid Peer angular2-meteor@0.5.5 wants @angular/core@2.0.0-rc.1

--- a/package.json
+++ b/package.json
@@ -26,15 +26,15 @@
     "test": "./run_tests.sh"
   },
   "peerDependencies": {
-    "@angular/core": "2.0.0-rc.1"
+    "@angular/core": "^2.0.0-rc.1"
   },
   "dependencies": {
     "angular2-meteor-polyfills": "^0.1.1"
   },
   "devDependencies": {
-    "@angular/core": "2.0.0-rc.1",
-    "@angular/common": "2.0.0-rc.1",
-    "@angular/compiler": "2.0.0-rc.1",
+    "@angular/core": "^2.0.0-rc.1",
+    "@angular/common": "^2.0.0-rc.1",
+    "@angular/compiler": "^2.0.0-rc.1",
     "es6-shim": "^0.35.0",
     "zone.js": "^0.6.12",
     "rxjs": "5.0.0-beta.6",

